genrl/deep/common/trainer.py

```diff
@@ -368,8 +368,7 @@ def train(self) -> None:
 
             # update params for DQN
             if self.agent.__class__.__name__ == "DQN":
-                if self.agent.replay_buffer.pos > self.agent.batch_size:
-                    self.agent.update_params()
+                self.agent.update_params(timestep)
```

Comment:
Also, why does DQN update on the basis of replay buffer pointer position rather than a timestep number as the others do?

Reply:
Okay then its easy will make it same as others.

Reply:
Awesome